packages/api/src/index.js

```diff
@@ -18,59 +18,78 @@ import {
 } from './maintenance.js'
 import { notFound } from './utils/json-response.js'
 import { nameGet, nameWatchGet, namePost } from './name.js'
+import { compose } from './utils/fn.js'
 
 const router = Router()
 router.options('*', corsOptions)
 router.all('*', envAll)
 
+router.get('*', withMode(READ_ONLY))
+router.head('*', withMode(READ_ONLY))
+router.post('*', withMode(READ_WRITE))
+router.delete('*', withMode(READ_WRITE))
+
+/**
+ * It defines a list of "middlewares" that need to be applied for a given authentication mode.
+ * Each value takes an endpoint handler and returns a "composed" version.
+ *
+ * @type {Object.<string, function(...any):any>}
+ */
```

Comment:
orphaned comment, plz delete
```suggestion
```

Reply:
@olizilla It might not be the best comment/annotation, but it is still relevant for the auth map, isn't it?

Reply:
oh sorry, i was skimming and i thought this was the comment for compose function!